src/NUnitFramework/mock-assembly/mock-nunit-assembly-4.5.csproj

```diff
@@ -39,6 +39,8 @@
     <DebugType>full</DebugType>
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>false</Prefer32Bit>
+    <Externalconsole>true</Externalconsole>
+    <Commandlineparameters>--test:NUnit.Tests.Assemblies.MockTestFixture</Commandlineparameters>
```

Comment:
There is a fair amount of noise in this PR. Just curious, were you using MonoDevelop?